src/components/Table/CollapseToggle.test.tsx

```diff
@@ -1,11 +1,15 @@
 import { CollapseToggle } from "src/components/Table/CollapseToggle";
-import { GridCollapseContext } from "src/components/Table/GridTable";
-import { noop } from "src/utils";
+import { RowState, RowStateContext } from "src/components/Table/RowState";
```

Comment:
I'm replacing the old `GridCollapseContext` that was value-/identity-based with a mobx-based context that makes it a lot easier to avoid re-renders.